hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -190,6 +190,20 @@ default boolean closeIsCooperative() {
     default void close(@Nullable Throwable error) throws Exception {
     }
 
+    /**
+     * Returns {@code true} if this instance is reusable, i.e. {@link #init}
+     * and {@link #close} can be called multiple times, possibly in parallel.
+     * <p>
+     * When a job is to be submitted, the job definition ({@link DAG} or
```

Comment:
serialization is described in class java doc, update it as well. It may be worthwile to add references to `isReusable` in `init` and `close` javadoc

Reply:
you still did not mention in class javadoc that serialization may be skipped and reading the class file from the top I would not expect that and so the description is incorrect.